MidCamp/midcamp
# User submitted sessions

## Description

- Satisfies "There should be a view of a user’s submitted sessions on their My Account page" on [MidCamp 2018 To-Do List](https://docs.google.com/document/d/14MwYh5nG2m5aKjrCPFaDy90m1WJ0bipTcBjYPx3yZW0/edit)
- Updates composer.json for strict validation
- Exports production config from today's `sql-sync`
- Adds user menu tab for Sessions view

## To test
- Run `composer install`.  Observe you no longer get the error for 
```
[Composer\Json\JsonValidationException]
  "./composer.json" does not match the expected JSON schema:
   - config.sort-packages : String value found, but a boolean is required
```
- `drush cim`
- Log in, view user profiles and observe they now show a tab for "User Submitted Sessions"

![image](https://user-images.githubusercontent.com/4048700/35311720-8255d81a-007d-11e8-97f8-91b3f08cf7bd.png)
